orsopy/fileio/__init__.py

```diff
@@ -1,4 +1,5 @@
 from orsopy.fileio.base import (ValueScalar, ValueRange, ValueVector, Comment,
                                 Person, Column)
+from orsopy.fileio.reduction import Software, Reduction
 
```

Comment:
I would always use the from .XXX import YYY form to make it possible/easy to include the package in other python projects directly.

Reply:
I have no opposition to this style, I just thought it would be clearer if the user could have a single import to get all of the data classes that are necessary.

Reply:
I normally have private classes and methods prefixed by _. I then add all public classes and functions to submodule.__all__. So the import would be from `orsopy.file import stuff`. For really important things you could add them to `orsopy.__all__`.

Reply:
I am tempted towards Artur's approach in the first instance and we can evolve in future based on how we use it.

Reply:
> I have no opposition to this style, I just thought it would be clearer if the user could have a single import to get all of the data classes that are necessary.

I agree, that it is nice to have all in the package. My point was about source of the import. What I meant was to replace
`from orsopy.XXX` by `from .XXX` so that it works if orsopy is a sub-packge of another python program.

Reply:
Ahh, that's a lot clearer